### PR TITLE
First improvements

### PR DESCRIPTION
**Pull request by filip-zyzniewski**
Best reviewed commit by commit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,59 @@
...
+  print the git commit SHA on stderr.
+
+- `-l <local_directory>`:
+  use a particular directory for a git checkout instead
+  of a tmpdir. If this directory is a git checkout, then
+  the `-r` option is ignored.
+
+- `-m <message>`:
+  use a particular git message content instead of
+  the ran command.
```
Comment (pbzdyl) [inline README.md]: What about:
```suggestion
  the ran command output.
```

**Review by pbzdyl — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (filip-taxamo) [reply README.md]: Command output is added with `-s`.

Comment (filip-zyzniewski) [reply README.md]: Does aec67687665d5f6100138ca460e207e1bba0c7fc make it clearer?

Comment (pbzdyl) [reply README.md]: Yes, thanks!

**Review by pbzdyl — APPROVED** [review]

**Pull request merged**